luaver

```diff
@@ -200,7 +225,7 @@ __luaver_download_and_unpack()
         read -r -p "${unpack_dir_name} has already been downloaded. Download again? [Y/n]: " choice
         case $choice in
             [yY][eE][sS] | [yY] )
-                __luaver_exec_command "rm -r ${unpack_dir_name}"
+                __luaver_exec_command "rm -fr ${unpack_dir_name}"
```

Comment:
I think `command rm -r "${unpack_dir_name}"` is enough to ignore alias.
Or should `__luaver_exec_command` prepend `command`?